asset/records.go

```diff
@@ -30,6 +34,71 @@ const (
 	LeafAssetID LeafTlvType = 11
 )
 
+// KnownAssetLeafTypes is a set of all known leaf types.
+var KnownAssetLeafTypes = fn.NewSet(
```

Comment:
Ideally we can set up some compile or `init` level thing to make sure that each time the set of types are updated, then this is as well.